virttest/qemu_vm.py

```diff
@@ -1828,6 +1828,18 @@ def sort_key(dev):
                               pci_bus=pci_bus)
                 iov += 1
 
+        #  Add general PCI Pass-through devices
+        for pci_pass_devs in params.objects("pci_pass_devices"):
```

Comment:
where does pci_pass_devices defined or configured ?

Reply:
Hello @balamuruhans!
There is an `cfg` example on my first comment of this conversation:
```
pci_pass_devices += " vfio1"
host_vfio1 = "05:0a.0"
# these are defaults:
#device_driver_vfio1 = "vfio-pci"
```
Please let me know, what kind of comment you want to add to python source code.

Reply:
> There is an cfg example on my first comment of this conversation:

@hpaluch-pil Agreed, let us place the params commented in guest-hw.cfg for using it from framework.

Reply:
```
variants:
    - @no_pci_assignable:
        pci_assignable = no
    - pf_assignable:
        pci_assignable = pf
        host_setup_flag = 3
        # device_name is optional. If device_name not set, it will try to get
        # first available pf. If device_name set, but the device is not
        # available, it will also try to get first available pf.
        device_name = eth1
        nettype = bridge
        pf_filter_re = "82576 Gigabit Network"
        mac_ip_filter = "inet (.\d+.\d+.\d+.\d+).*?ether (.\w+:\w+:\w+:\w+:\w+:\w+)"
        # PCI Passthrough for general PCI cards (we use it for our relay switching cards).
        # pci_passthrough_devices = " vfio1"
        # host_vfio1 = "05:0a.0"
        RHEL.6, RHEL.5:
            mac_ip_filter = "HWaddr (.\w+:\w+:\w+:\w+:\w+:\w+)\s+?inet addr:(.\d+\.\d+\.\d+\.\d+)"
        mac_changeable = "yes"
        unattended_install:
            mac_changeable = no
```